versioning/BRM/extentmap.cpp

```diff
@@ -1946,22 +1899,12 @@ void ExtentMap::grabEMIndex(OPS op)
     }
 }
 
-
 void ExtentMap::releaseEMEntryTable(OPS op)
 {
     if (op == READ)
-    {
         fMST.releaseTable_read(MasterSegmentTable::EMTable);
-    }
     else
     {
-        /*
-           Note: Technically we should mark it unlocked after it's unlocked,
```

Comment:
Why do you think this comment is irrelevant now?

Reply:
Removed by accident, thanks.